ledger/metrics/src/main/scala/com/daml/metrics/OpenTelemetryMeterOwner.scala

```diff
@@ -20,7 +25,26 @@ case class OpenTelemetryMeterOwner(enabled: Boolean, reporter: Option[MetricsRep
   override def acquire()(implicit
       context: ResourceContext
   ): Resource[Meter] = {
-    val meterProviderBuilder = SdkMeterProvider.builder()
+    val meterProviderBuilder = SdkMeterProvider
+      .builder()
+      .registerView(
+        InstrumentSelector
+          .builder()
+          .setType(InstrumentType.HISTOGRAM)
+          .setName(new Predicate[String] {
+            override def test(t: String): Boolean = t.endsWith("duration.ms")
+          })
+          .build(),
+        View
+          .builder()
+          .setAggregation(
+            Aggregation.explicitBucketHistogram(
```

Comment:
Can you possibly share the reference from where this bucket definition comes, please?

Reply:
The buckets are defined based on the data/granularity that we expect to encounter therefore there is no reference

Reply:
We have default buckets defined e.g. for the [Python](https://github.com/prometheus/client_python/blob/master/prometheus_client/metrics.py#L544) and [Go](https://github.com/prometheus/client_golang/blob/main/prometheus/histogram.go#L265) Prometheus clients that we could leverage.
I don't have a strong preference (i.e., we may keep the current bucketing). However, from a practical perspective I'm not sure if it makes sense to capture different buckets above 10s latency (i.e., do we have calls that we expect to take that long?).

Reply:
That's a good point, it was mostly for certain scenarios where that could be useful. But I'll default to the go defined ones for now, it makes more sense...and we can change it in the future.

Reply:
Will cover that in a separate PR